CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Fixed
 * Added better comparator for `realm_user_t` and `realm_flx_sync_subscription_t` when using `realm_equals`.(Issue [#5522])(https://github.com/realm/realm-core/issues/5522).
+* Removed scheduler argument to the C API `realm_*_add_notification_callback` functions, because it wasn't actually used. (PR [#5541](https://github.com/realm/realm-core/pull/5541)).
```

Comment:
Probably this should go behind "breaking changes", the SDKs will have to adjust their code in order to cope with this change.